client/app/boot.ts

```diff
@@ -1,3 +1,8 @@
 import {bootstrap}    from 'angular2/platform/browser';
 import {AppComponent} from './app.component';
-bootstrap(AppComponent);
+import {HTTP_PROVIDERS}    from 'angular2/http';
```

Comment:
This stuff actually should be in client/app/app.component.ts, as that is the all encompassing component of the site.